sonar-plugin-api/src/main/java/org/sonar/api/profiles/ProfileExporter.java

```diff
@@ -45,6 +47,11 @@ protected ProfileExporter(String key, String name) {
     this.name = name;
   }
 
+  /**
+   * Export activated rule from a quality profile to a writer
+   *
+   * Note that the quality profile can contains some rules from other plugins, and it should not fail in this case.
```

Comment:
Note that the quality profile can contain some rules from other plugins. It should not fail in this case.